WebJobs.Script.sln

```diff
@@ -352,6 +352,10 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "HttpTrigger-HostJsonRetry",
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Azure.WebJobs.Script.Tests.Benchmarks", "test\Benchmarks\Microsoft.Azure.WebJobs.Script.Tests.Benchmarks.csproj", "{09D16953-A048-4E6B-B366-1E0D7E5EF86E}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "WebJobs.Script.Analyzers", "src\WebJobs.Script.Analyzers\WebJobs.Script.Analyzers.csproj", "{FDCD62D4-051E-45D9-8794-BE9A5CABB5D1}"
```

Comment:
Makes sense, but do we also want to move the projects within the folder structure? Would it be more appropriate to put them under `tools/analyzers` rather than `src`?

Reply:
`src` still makes sense for this. You'd also have a project folder under that, scoped to that project only.